.github/workflows/publish-edge.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Edge
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  publish-edge:
+    name: Publish Edge
+    runs-on: ubuntu-16.04
```

Comment:
Might be worth bumping to at least 18.04, 16.04 is getting old 👴

Reply:
I'll definitely look into it.  If I recall correctly, the cypress action had issues with 18.04 at the time.  Hopefully that's resolved.